db/sql/revert/buildings_reference/huts_and_protected_areas.sql

```diff
@@ -0,0 +1,19 @@
+-- Revert nz-buildings:buildings_reference/huts_and_protected_areas from pg
+
+BEGIN;
+
+-- Drop columns from building_reference_log.
+
+ALTER TABLE buildings_reference.reference_update_log DROP protected_areas;
```

Comment:
Use `DROP COLUMN` instead of just `DROP` - while this defaults to `COLUMN`, it makes it more explicit to just include the word anyway.